terraform/070-id-sync/README.md

```diff
@@ -40,6 +40,7 @@ store.
 
 - `email_service_assertValidIp` - Whether or not to assert IP address for Email Service API is trusted
 - `id_broker_assertValidIp` - Whether or not to assert IP address for ID Broker API is trusted
+- `alerts_email` - Who to send alerts to about sync problems
 - `notifier_email_to` - Who to send notifications to about sync problems (e.g. users lacking email addresses)
```

Comment:
What's the difference between the purpose of `alerts_email` and `notifier_email_to`?

Reply:
Excellent question.
`notifier_email_to` is for notifications to HR, currently it appears to only be used for missing email notifications
`alerts_email` is for other errors and exceptions (exceptions other than 400-level errors and `EmailServiceClientException`)

Reply:
It may be worth clarifying that purpose in the comment about each here, so that it's not just tribal knowledge. Just a thought.

Reply:
Good suggestion. I did that on `idp-template-terraform` and will do it here when I merge to `main`.